bazel_tools/haskell-ghci-grpc.patch

```diff
@@ -1,9 +1,66 @@
 On MacOS GHCi fails to load grpc fat_cbits statically due to duplicate symbols.
+diff --git a/haskell/private/path_utils.bzl b/haskell/private/path_utils.bzl
+index 3956164a..037cc95a 100644
+--- a/haskell/private/path_utils.bzl
++++ b/haskell/private/path_utils.bzl
+@@ -135,6 +135,44 @@ def make_library_path(hs, libs, prefix = None):
+ 
+     return join_path_list(hs, set.to_list(r))
+ 
++def symlink_dynamic_library(hs, lib, outdir):
```

Comment:
This function was removed from upstream rules_haskell. We still need it for `fat_cbits`, so it's added to this patch.